apps/console/src/components/pages/protected/evidence/table/columns.tsx

```diff
@@ -15,6 +15,12 @@ type TGetEvidenceColumnsProps = {
 
 export const getEvidenceColumns = ({ userMap }: TGetEvidenceColumnsProps) => {
   const columns: ColumnDef<Evidence>[] = [
+    {
+      accessorKey: 'displayID',
+      header: 'ID',
+      size: 120,
+      cell: ({ row }) => <div className="text-muted-foreground">{row.original.displayID || row.original.id}</div>,
+    },
```

Comment:
`DisplayID` is a different field/id, this should consistently be `id` in this case
```suggestion
    {
      accessorKey: 'id',
      header: 'ID',
      size: 120,
      cell: ({ row }) => <div className="text-muted-foreground">{row.original.id}</div>,
    },
```